app/views/searches/search.html.erb

```diff
@@ -0,0 +1,72 @@
+<div class="row">
+  <% if !@provider.nil? %>
+  <div class="jumbotron">
+    <h1>
+      <%= @provider.capitalize %> Search Results
+    </h1>
+  </div>
+  <% else%>
+    <div class="jumbotron">
+      <h1>Please select Twitter or Vimeo</h1>
+    </div>
+<!-- </div> -->
+<% end %>
+<% if !@results.nil? %>
+  <% @results.in_groups_of(3, false) do |group| %>
+    <div class="row">
+      <% group.each do |result| %>
+        <div class="col-sm-4 col-md-4 creator-col">
+          <% if @provider == "twitter" %>
+            <% picture = result.profile_image_uri if !result.profile_image_uri.nil? %>
+          <% elsif @provider == "vimeo" %>
+            <% picture = result["pictures"]["sizes"][2]["link"] if !result["pictures"].nil? %>
+            <% picture = "vimeo_default_two.png" if result["pictures"].nil? %>
+          <% end %>
+          <%= image_tag(picture, class: "creator-image") %>
+          <h4 class="creator-name">
+            <%= result.name if @provider == "twitter" %>
+            <%= result["name"] if @provider == "vimeo" %>
+          </h4>
+           <% if @provider == "twitter" %>
+             <% if !(@current_user.twitter_follow?(result)) %>
+               <%= button_to "Follow", {controller: "creators",
+                                        action: "follow",
+                                        name: result.screen_name,
+                                        provider: @provider,
+                                        bio: result.description,
+                                        profile_pic: result.profile_image_uri,
+                                        uid: result.id},
```

Comment:
This seems like a lot of things to pass in from the view. Do you need to persist all of this info?